sdk/avalon_sdk/work_order/work_order_request_validator.py

```diff
@@ -70,104 +40,183 @@ def validate_parameters(self, params):
         if len(params) == 0:
             return False, "Empty params in the request"
 
-        key_list = []
-        for key in params.keys():
-            if key not in self.__param_key_map.keys():
-                return False, "Invalid parameter {}".format(key)
+        schema = {
```

Comment:
I would like to suggest to keep the schema in a separate file. JSON schema file name can be in format APIname.jsonschema (eg: WorkOrderSubmit.jsonschema) and then read this schema file from python code when we need to validate the parameters. This will keep schema definitions separate and will be easier to maintain.

Reply:
Done